### ISSUE TITLE
Adjusting/ updating compass (locked/protected)

### ISSUE THREAD

**Issue by Klaranth** [pandorabox-io/in-game#142]
UncleSam ; ThinkSome ; SX ; Gorlock 

Protected or locked compass so only owner can use it.
Store in protected or locked chest or backpack would suffice.

Perhaps no teleport zones?
I mean in a way that prevents anyone not in protection to use any teleports (missions, compass, tp requests, whatever that does not require active nodes)

UncleSam
What if we lose a compass, anyone having it can access a secure private area. E.g.  if we die on a protected area with vacuum tubes.

SX 
I mean if you do also store in backpack when not using it and only take it out immediately before using and verifying that location is safe then I don't think you really can die, with that super important compass in your inventory. Some way to disallow teleport to some areas could be useful but I don't think restricting compass is that useful.

Gorlock
No compass area, with magnetic perturbations?

SX 
For generic teleport restriction I could see many more use cases (minigames for one) and it would also allow restricting compass use a lot more if needed.
(And it would come without adding another item and relatively complicated metadata stuff for it)
Basically locked/protected compass has few big problems: protected will be very hard to maintain and it would not be complete solution alone (very easy to get around restriction)

I would make those only no-teleport-in, not areas where you cannot teleport out. complete no teleport at all zones probably should not be allowed without some trusted player privileges
That is if you program your own mission wand or compass you would not be able to use it without being in protection access list.
But while in area you could still use /home and /spawn and also /tpr to players who are not in no teleport zone
That would probably be both safe and allow enough restrictions for most acceptable use cases.

**Comment by S-S-X** [comment]
No teleport zones could be useful but I think request as is is not useful enough, restricting compass also use wont really help much with privacy.

Before implementing something like this there should be some real use cases for it.
Some minigame setups probably could be a thing to enforce rules in game area but also there it would be good to have real use case that shows what has to be done and why it has to be done.
